user/incus/files/sysusers.conf

```diff
@@ -0,0 +1,4 @@
+# Create incus system users
+
+u incus - "incus user" /var/lib/incus /usr/bin/nologin
+g incus-admin -
```

Comment:
if the group is easily usable with an underscore, name this _incus / _incus_admin (since the above daemon should then be runnable with --group _incus_admin)

Reply:
(the reason for this is we just have _-prefixed system groups/users where possible, but it isn't always. if it then tries to use a hardcoded group name from the code then sometimes it's patchable but if it's too much effort what you put already is ok)